src/pages/MovieInfo/MovieInfo.js

```diff
@@ -1,44 +1,43 @@
 import React, { Component, Fragment } from "react";
 import { trackPromise } from "react-promise-tracker";
-import LoadingIndicator from "../../components/LoadingIndicator/LoadingIndicator";
-
+import LoadingIndicator from "../../components/LoadingIndicator";
+import FavBtn from "../../components/FavBtn";
+import ErrorMessage from "../../components/ErrorMessage";
+import getMovie, { PLOT_LENGTH } from "../../api/getMovie";
 import "./MovieInfo.sass";
 
 class MovieInfo extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      movie: ""
-    };
+  state = {
+    movie: "",
```

Comment:
I would use `null` for example, as string could be misleading that this will be just a string